packages/apps/plugins/plugin-markdown/src/components/DocumentEditor.tsx

```diff
@@ -58,17 +87,32 @@ const DocumentEditor = ({ document: doc, extensions: _extensions = [], ...props
     };
   }, [doc]);
 
-  if (!doc.content) {
```

Comment:
this wasn't necessary?

Reply:
I moved this to the surface renderer, we shouldn’t be returning `null` so late and we can check this value a lot sooner.